spec/requests/memberships_controller_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+describe "/memberships", type: :request do
+  let(:user) { create(:user) }
+  let(:group) { create(:group) }
+  let(:role) { :group_admin }
```

Comment:
Could we rename this to make it easier to separate out the logged-in-user's role from the membership-being-deleted's role

```suggestion
  let(:logged_in_user_role) { :group_admin }
```